application/core/MY_Controller.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * core/MY_Controller.php
+ *
+ * Default application controller
+ *
+ * @author		JLP
+ * @copyright           2010-2013, James L. Parry
+ * ------------------------------------------------------------------------
+ */
+class Application extends CI_Controller {
+
+	protected $data = array();	  // parameters for view components
+	protected $id;			  // identifier for our content
+
+	/**
+	 * Constructor.
+	 * Establish view parameters & load common helpers
+	 */
+
+	function __construct()
+	{
+		parent::__construct();
+		$this->data = array();
+		$this->data['title'] = 'Assignment 1: GameBots-05';	// our default title
+		$this->errors = array();
+		$this->data['pageTitle'] = 'welcome';   // our default page
+	}
+
+	/**
+	 * Render this page
+	 */
+	function render()
+	{
+		$this->load->library('parser');

```

Comment:
@josephwithay We don't need this line anymore now that you have put it into autoload